vowpalwabbit/parse_args.cc

```diff
@@ -1143,7 +1148,12 @@ void parse_output_model(options_i& options, VW::workspace& all)
   if (!all.final_regressor_name.empty() && !all.logger.quiet)
     *(all.trace_message) << "final_regressor = " << all.final_regressor_name << endl;
 
-  if (options.was_supplied("invert_hash")) all.hash_inv = true;
+  if (options.was_supplied("invert_hash")) { all.hash_inv = true; }
+  if (save_resume)
+  {
+    logger::errlog_warn("--save_resume flag is deprecated -- learning can now continue on saved models by default!");
```

Comment:
Small thing, I don't think the ! is necessary.
```suggestion
    logger::errlog_warn("--save_resume flag is deprecated -- learning can now continue on saved models by default.");
```